edgePy/edgePy.py

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
Move the CLI script to `edgePy/scripts` and add it to the `setup.py` key:value  for script install.

See:

http://python-packaging.readthedocs.io/en/latest/command-line-scripts.html